contents/blog/hotjar-for-mobile-apps-alternatives.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: 'Hotjar alternatives for mobile apps (iOS & Android)'
```

Comment:
Hmm not sure I fully agree:
1. I think Google understands the different between "hotjar alternatives" and "hotjar alternatives for mobile".
2. Here are keywords I found volume for:
    - hotjar alternatives for mobile apps (40/m)
    - Hotjar for mobile apps  (210/m)
    - Hotjar for mobile (10, ahrefs)

So having "alternatives" in the title feels like it will help us rank for some riches.

In any case, if you still feel we should avoid it, here are some alternatives I came up with:
- Hotjar replacements for mobile apps (iOS & Android) – I dont really like this since "replacement" implies it already existed
- Hotjar-like tools for mobile apps (iOS & Android) – reads a bit better but Im worried it wont rank for the above searches
- Hotjar equivalents for mobile apps (iOS & Android)

Reply:
Experience has taught me otherwise! You'd be amazed how dumb / annoying google can be about stuff like this. Best to play it safe. 

I've put in "What's the best Hotjar for mobile apps?" as the title, and added a meta description that mentions the frameworks.